package.json

```diff
@@ -2,8 +2,8 @@
   "name": "electron-packager",
   "version": "17.1.2",
   "description": "Customize and package your Electron app with OS-specific bundles (.app, .exe, etc.) via JS or CLI",
-  "main": "src/index.js",
-  "types": "src/index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/types.d.ts",
```

Comment:
thought: if we're going to be switching to TypeScript anyway, perhaps in a follow-up PR we should add ESM support.

Reply:
Sounds great!